utilities/rocAL/rocAL_performance_tests_with_depth/rocAL_performance_tests_with_depth.cpp

```diff
@@ -127,16 +127,16 @@ int test(int test_case, const char* path, int rgb, int gpu, int width, int heigh
 
 
     /*>>>>>>>>>>>>>>>>>>> Graph description <<<<<<<<<<<<<<<<<<<*/
-    RocalImage inputImage;
-    RocalImage image0;
-    RocalImage image0_b;
+    RocalTensor input_image;
```

Comment:
rename all image variable names to tensor

Reply:
Renamed